botorch/utils/gp_sampling.py

```diff
@@ -197,44 +199,59 @@ def _get_weights(
         return weights
 
     def forward(self, X: Tensor) -> Tensor:
-        """
-        Get fourier basis features for the provided inputs.
-        Note that if `sample_shape` has been passed, then the rightmost
-        subset of the batch shape of the input should be `sample_shape`.
+        """Get fourier basis features for the provided inputs.
+
+        Note that the right-most subset of the batch shape of `X`
```

Comment:
```suggestion
        Note that this will error unless the right-most subset of the batch shape of `X`
```
(making it clear the user doesn't need to check manually)